ee/packages/pdf-worker/src/strategies/ChatTranscript.spec.ts

```diff
@@ -60,4 +60,10 @@ describe('Strategies/ChatTranscript', () => {
 			result.t('invalidKey');
 		}).toThrow('Translation not found for key: invalidKey');
 	});
+
+	it('should parse the system message', () => {
```

Comment:
I guess this test case doesn't makes any sense here, since this service/class is responsible for just rendering the provided data, it doesn't contain the list of valid system messages.

Maybe we should test it on omnichannel-services where this parsing and logic happens.

Reply:
Implemented unit tests for `OmnichannelTranscript` class and covered this case over there